src/main/java/org/stellar/sdk/Signer.java

```diff
@@ -4,12 +4,15 @@
 import org.stellar.sdk.xdr.SignerKeyType;
 import org.stellar.sdk.xdr.Uint256;
 
+import static com.google.common.base.Preconditions.checkArgument;
 import static com.google.common.base.Preconditions.checkNotNull;
 
 /**
  * Signer is a helper class that creates {@link org.stellar.sdk.xdr.SignerKey} objects.
  */
 public class Signer {
+    public static final int SIGNED_PAYLOAD_MAX_PAYLOAD_LENGTH = 64;
```

Comment:
Ah yeah, I think this belongs above in `SignedPayloadSigner`

Reply:
ok, I had located validation here in the [Signer.signedPayload(SignedPayloadSigner)](https://github.com/stellar/java-stellar-sdk/pull/413/files#diff-35deaf3ac00513f2a942ad2b4eb0415b4461402d49365bd38d312ec283e53eb3R84) factory method which constructs the xdr SignerKey for payload signer from the payload.  The intent on `SignedPayloadSigner` was to be data transfer object, no logic, to your point, it could take on that role also of validation, but thought was keeping that logic closer to when it's used in SignerKey, let me know if you want to discuss more, thanks!